### ISSUE TITLE
monitor + visualization

### ISSUE THREAD

**Issue by dewiweb** [philhartung/aes67-monitor#6]
Hi Philipp,
Do you plan to integrate visualization in monitor app?

**Comment by philhartung** [comment]
Yes I do! That's a thing that is pretty high on the todo list, though I didn't have any time to work on this project the last months  :(

**Comment by philhartung** [comment]
You might want to check out https://github.com/philhartung/aes67-visualization in the meantime. It's a little tool I developed to launch ffmpeg audio visualizations from SDP files.
